src/lib/util.ts

```diff
@@ -167,6 +167,10 @@ export function createModlogEmbed(embedData: EmbedData, modlog: PunishmentLog, m
 	return new EmbedBuilder(embedData)
 }
 
+export function formatNumberToLocaleString(number: number, getString: GetStringFunction) {
+	return number.toLocaleString(getString("region.dateLocale", { file: "global" }))
+}
```

Comment:
IMO you can remove this function and instead change parseToNumberString to use toLocaleString if the number is smaller than 1k